Observing-Interface-Setup/getting-started-nublado.rst

```diff
@@ -0,0 +1,170 @@
+
+    Screenshot of nublado launch page.
+
+If you now click the "Launch Server" button, you will be taken to the :ref:`Server Options <Observing-Interface-Getting-Started-Nublado-Server-Options>` page.
+
+.. figure:: ./_static/nublado-server-options.png
+    :name: Observing-Interface-Getting-Started-Nublado-Server-Options
+
+    Screenshot of nublado server options page.
+
+In this page, you can now select the image and the resources you need for your work.
+
+  - The "image" selection specify which version of the science pipelines to load (the observatory control packages are always the same).
+
+  - The "size" option specify the amount of computing resources that will be made available for your jupyter-lab instance.
+
+In most cases, you will want to select the "Recommended" image and "Medium" size.
+If you plan to do more computing intensive tasks you may also select a "Large" image.
+
+Once you selected your "Image" and "Size" you can click in the big orange "Start" button.
+You will be redirected to a :ref:`loading page <Observing-Interface-Getting-Started-Nublado-Loading>`.
+
+.. figure:: ./_static/nublado-startup.png
+    :name: Observing-Interface-Getting-Started-Nublado-Loading
+
+    Screenshot of nublado loading page.
+
+In general, sections will load pretty quickly, though they may take longer if the "Image" you selected is still not available in the server.
+
+Once loading is completed you will be redirected to a :ref:`jupyter-lab section <Observing-Interface-Getting-Started-Nublado-JupyterLab>`.
+
+.. figure:: ./_static/nublado-jupyter-lab.png
+    :name: Observing-Interface-Getting-Started-Nublado-JupyterLab
+
+    Screenshot of jupyter-lab section after successful login.
+
+To start a new notebook for data analysis and/or interaction with the observatory control system, click in the "LSST" icon in the "Notebook" section.
+If you select "Python 3", it will create a bare python 3 kernel which does not have access to the science pipelines or the observatory control algorithms.
+
+Alternatively, you can clone the `ts_notebooks`_ repository, which contains a collection of notebooks from users as well as examples and procedures.
+To do that, open a terminal by clicking in the "Terminal" icon in the "Other" section.
+This will open a linux terminal on your browser, that is running on the remote server in the environment you choose to operate.
+From here you can execute regular linux terminal commands.
+
+.. _ts_notebooks: https://github.com/lsst-ts/ts_notebooks
+
+To clone the `ts_notebooks`_ repository do;
+
+  .. prompt:: bash
+
+      cd ~/notebooks
+      git clone https://github.com/lsst-ts/ts_notebooks.git
+
+You can than navigate to the location where the repository was cloned using the file browser on the left hand side of the screen, clicking on "notebooks" and than "ts_notebooks".
+There are useful notebooks to start with in the "examples" and "procedures" folders.
+
```

Comment:
I think this is a good place to put a sentence about setting up environments and link to your other page.